test/models/deploy_metrics_test.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+require_relative '../test_helper'
+
+SingleCov.covered!
+
+describe DeployMetrics do
+  include GitRepoTestHelper
+
+  let(:date_time_now) { Time.now }
+  let(:project) { projects(:test) }
+  let(:stage) { stages(:test_staging) }
+  let(:user) { users(:deployer) }
+  let(:deploy) { deploys(:succeeded_test) }
+  let(:deploy_metrics) { DeployMetrics.new(deploy) }
+
+  describe "#cycle_time" do
+    describe "production deployment with no staging deployment" do
+      let(:deploy) { deploys(:succeeded_production_test) }
+      let(:deploy_metrics) { DeployMetrics.new(deploy) }
+      before do
+        deploy.updated_at = date_time_now + 25
+        deploy.changeset.expects(:pull_requests).
+          returns([stub("commit 1", created_at: date_time_now), stub("commit 2", created_at: date_time_now + 10)])
+      end
+
+      it "must return pr_production cycle time" do
```

Comment:
can simplify this to:
```suggestion
      it "returns pr_production cycle time" do
```